security/pkg/pki/testdata/crl/cert.sh

```diff
@@ -0,0 +1,178 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

Since not all environments place bash there (looking at you... Apple...), better practice to use this.